resources/tentative/todomvc/src/components/todo-list/todo-list.template.js

```diff
@@ -0,0 +1,12 @@
+const template = document.createElement("template");
+const globals = "src/styles/global.css";
+const styles = "src/styles/todo-list.css";
+
+template.id = "todo-list-template";
+template.innerHTML = `
+    <link rel="stylesheet" href="${globals}" />
```

Comment:
> I believe using `<link rel=stylesheet>` in an element's template may cause a fetch of the stylesheet for each instance of the element. With proper caching this may still be reasonably fast but as I understand it the platform's intended primitive for this use case is constructible stylesheets.

Browser would cache the content referenced by link element just fine.